src/overlays/actors/ovl_En_Ru/z_en_ru.c

```diff
@@ -76,34 +86,273 @@ static DamageTable D_80A39488 = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
 
-#endif
+static AnimationInfoS sAnimations[] = {
+    { &gAdultRutoIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
+    { &gAdultRutoIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoRaisingArmsUpAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoCrossingArmsAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoLookingDownLeftAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoIdleHandsOnHipsAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoHeadTurnDownLeftAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+    { &gAdultRutoSwimmingUpAnim, 1.0f, 0, -1, ANIMMODE_LOOP, -4 },
+};
+
+// in PostLimbdraw, converts limbIndex to bodyPartsPos index
+static s8 sBodyPartPosIndices[] = {
+    -1, -1, 12, 13, 14, -1, 9, 10, 11, -1, 0, 6, -1, -1, 7, 8, 2, -1, -1, 3, 4, 2, 1, 0
+};
+
+static s8 sRuBodyParts[] = {
+    0, 0, 0, 0, 3, 4, 0, 6, 7, 0, 9, 10, 0, 12, 13, 0,
+};
+
+static u8 sRuShadowSizes[] = {
+    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+};
```

Comment:
```suggestion
static s8 sBodyPartPosIndices[] = {
    -1, -1, 12, 13, 14, -1, 9, 10, 11, -1, 0, 6, -1, -1, 7, 8, 2, -1, -1, 3, 4, 2, 1,
};

static s8 sRuBodyParts[] = {
    0, 0, 0, 0, 3, 4, 0, 6, 7, 0, 9, 10, 0, 12, 13,
};

static u8 sRuShadowSizes[] = {
    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
};
```
Looking at how they are used, these extra 0's are padding. `sBodyPartPosIndices` should have the same size as `RU2_LIMB_MAX`, while both `sRuBodyParts` and `sRuShadowSizes` should match the size passed to `SubS_GenShadowTex`.